data-model/src/main/scala/za/co/absa/enceladus/model/conformanceRule/package.scala

```diff
@@ -153,7 +157,8 @@ package object conformanceRule {
 
   object MappingConformanceRule {
     // attributeMappings property has key's with dot's that mongo doesn't accept; this symbol is used to replace the dots
-    val DOT_REPLACEMENT_SYMBOL: Char = '^'
+    final val DOT_REPLACEMENT_SYMBOL: Char = '^'
+    final val DEFAULT_OVERRIDE_MAPPING_TABLE_OWN_FILTER: Boolean = false
```

Comment:
I don't think this is the proper letter casing in Scala for constants. I believe `CapitalizedCamelCase` should be used

Reply:
I agree. But I adhered to the existing constant. On the other hand, why not to change both of them... 🤔

Reply:
I also had the same first impression: [here](https://docs.scala-lang.org/style/naming-conventions.html)